hwy/ops/x86_128-inl.h

```diff
@@ -5592,6 +5592,24 @@ HWY_API Vec128<float, N> ConvertTo(Simd<float, N, 0> /* tag */,
   return Vec128<float, N>{_mm_cvtepi32_ps(v.raw)};
 }
 
+template <size_t N>
+HWY_API Vec128<float, N> ConvertTo(HWY_MAYBE_UNUSED Simd<float, N, 0> dd,
```

Comment:
Optional: we usually use df float float, perhaps dd for double. Might be nice to rename to df.